advocacy_docs/partner_docs/ImpervaDataSecurityFabric/03-SolutionSummary.mdx

```diff
@@ -0,0 +1,16 @@
+---
+title: 'Solution Summary'
+description: 'Brief explanation of the solution and its purpose'
+---
+
+The Imperva Data Security Fabric (DSF), previously Imperva SecureSphere, system protects data with a hybrid security solution that is usable by all data types.  Imperva Data Security Fabric can help organizations reduce security risks from inside and outside sources, whether it be a careless mistake or malicious intent.
```

Comment:
Change first sentence to: 

"Imperva Data Security Fabric (DSF), previously known as Imperva SecureSphere, protects data with a hybrid security solution that is usable by all data types."